e2e/tests/regression/test_backup.robot

```diff
@@ -106,3 +107,29 @@ Test Incremental Restore
     And Delete pod 0
     And Delete persistentvolumeclaim for volume 3
     And Delete persistentvolume for volume 3
+
+Test Uninstallation With Backups
+    [Tags]    uninstall
+    [Documentation]    Test uninstall Longhorn with normal and failed backups
+    Given Set setting deleting-confirmation-flag to true
+    And Create volume 0 with    dataEngine=v1
+    And Attach volume 0
+    And Wait for volume 0 healthy
+    And Write data 0 to volume 0
+    And Create volume 1 with    dataEngine=v2
+    And Attach volume 1
+    And Wait for volume 1 healthy
+    And Write data 1 to volume 1
+
+    # create failed backups by resetting backupstore and create backups without available backup targets
+    And Reset Backupstore
+    And Create backup 0 for volume 0 without waiting for completion
+    And Create backup 1 for volume 1 without waiting for completion
```

Comment:
Is it possible to check failed backup here?  Because in `self.backup.create(volume_name, backup_id, wait)` will return if `wait` is False. Beside this, no other questions.

Reply:
Refined. Thank you!